### PR TITLE
Added project files

### PR DESCRIPTION
**Pull request by ankitt26**
In this branch i did -

- [ ]  Add index.html  file.
- [ ]  Add styles.css  file.
- [ ]  Add linters.yml file.
- [ ] Add readme.md file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,219 @@
...
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [First-microverse-project] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
```
Comment (Qoosim) [inline README.md]: - [x] Kindly remove this line. It is there to inform you what to do.

File: README.md [hunk 2]
```diff
@@ -1 +1,219 @@
...
+
+## 🛠 Built With <a name="built-with"></a>
+
+1- HTML.
+2- CSS.
+3- Linters
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
```
Comment (Qoosim) [inline README.md]: - [x] Same thing goes for this too

File: README.md [hunk 3]
```diff
@@ -1 +1,219 @@
...
+    <li><a href="https://reactjs.org/">CSS</a></li>
+    <li><a href="https://reactjs.org/">Javascript</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
```
Comment (Qoosim) [inline README.md]: - [x] Same thing here

File: README.md [hunk 4]
```diff
@@ -1 +1,219 @@
...
+
+> coming soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
```
Comment (Qoosim) [inline README.md]: - [x] Same thing here

File: README.md [hunk 5]
```diff
@@ -1 +1,219 @@
...
+
+### Deployment
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
```
Comment (Qoosim) [inline README.md]: - [x] Same thing here

File: README.md [hunk 6]
```diff
@@ -1 +1,219 @@
...
+- Twitter: [@twitterhandle](https://twitter.com/ankit26k)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ankit26k/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
```
Comment (Qoosim) [inline README.md]: - [x] Same thing here.

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @ankitt26, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ankitt26** [comment]
Modified Readme.md file

**Review by Shedrack-Sunday — APPROVED** [review]
# STATUS: APPROVED ♻️♻️
Hi @ankitt26 

To highlight 👍🏽
✔️ All suggested changes made
✔️ No linter errors

Your project is complete! There is nothing else to say other than... it's time to merge it 🚀
Congratulations! 🎉

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**